### ISSUE TITLE
How can I make the query example fields only expose fields which are exposed on the type?

### ISSUE THREAD

**Issue by hayfever** [anvilco/spectaql#11]
I'm experiencing an issue where I have a query documented with only one field exposed on it's return type, yet all fields on the return type are shown in the query example. The rest of the fields are appropriately hidden in the type example. My expectation is that because I've only exposed one field in the type itself that the return type for the query would also only show that one field. Is this intended or is there a configuration setting I'm overlooking?

**Comment by newhouse** [comment]
Ah, I think I know what you're saying and what is going on. If the information is not sensitive, I'd love a couple of screen shots?

All of the "hiding" takes place in the `JSON Schema` representation of your schema, and drives the flow of the resulting documentation. However, the examples stuff plays off of the `GraphQL Schema` built by GraphQL utils...I'm sure the problem is somewhere in there where the hidden things are not respected. I will have a look.

**Comment by hayfever** [comment]
Sure, here's a contrived example:

```
{
  "SCALAR": {
    "Boolean": {
      "documentation": {
        "documented": true
      }
    }
  },
  "OBJECT": {
    "Feature": {
      "documentation": { "documented": true },
      "fields": {
        "enabled": {
          "documentation": { "documented": true }
        }
      }
    },
    "Query": {
      "fields": {
        "features": {
          "documentation": { "documented": true }
        }
      }
    }
  }
}
```

<img width="1511" alt="Screen Shot 2021-03-23 at 8 56 48 PM" src="https://user-images.githubusercontent.com/3801926/112242898-a3330400-8c1a-11eb-8b2e-c1841c013283.png">

I would expect that the query example would not show `name` in the example.

I could also try to tackle soon, I was recently messing with dociql and I know this is something in there for this based off of the `select` option.

**Comment by newhouse** [comment]
Thank you so much for the examples and screenshot - super helpful.

I'm assuming based on looking at this that your `config.yml` has set all the `xxxDocumented`s and `xxxDocumentedDefault`s  all set to `false`...and the image is what I was expecting to see.

Hopefully that means I'm on the right track on what the probem is, as well...going to dig in now.

**Comment by newhouse** [comment]
Fix is merged. Also published `0.0.8`, but I don't think you're using NPM...thanks again for your help!
